Cookbook-ios/Utils/Features.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  Features.swift
+//  Cookbook-ios
+//
+//  Created by Carolina Arcos on 1/8/20.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+
+enum Features: CaseIterable {
+    case carousel
+    case codeScanner
+    case test
+    case test2
```

Comment:
Are `test` and `test2` necessary?

Reply:
I added them to test the carousel. They will be removed when other functionalities are added